packages/cli/test/ci-collect.test.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const {runCLI} = require('./test-utils.js');
+
+describe('collect', () => {
+  const fixturesDir = path.join(__dirname, 'fixtures');
+  const rcFile = path.join(fixturesDir, 'lighthouserc.json');
+
+  let tempDir;
+  beforeAll(() => {
+    tempDir = fs.mkdtempSync(path.join(os.tmpdir(), 'lighthouserc-'));
```

Comment:
FYI I added a `withTmpDir` command to utils that auto cleans up everything

https://github.com/GoogleChrome/lighthouse-ci/blob/b26fe670994d81ab3a5931bb2bcc98cc25f0f207/packages/cli/test/test-utils.js#L47-L51

example usage:

```js
it('should test', async () => {
  await withTmpDir(dir => {
    // do something with the dir
  })
})
```

Reply:
Cool, using it now `withTmpDir`. 

By the way we should `try... finally` that function. If the test fails the dir won't be deleted.

Reply:
sg! the cleanup isn't *super* critical since it's in a tmpdir anyhow but definitely a great point if it's worth that ugly indentation it's worth a try finally :)